package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "toggle-gifs",
+  "version": "2.0.0",
+  "description": "Start/stop GIF animations through a keyboard shortcut or by clicking them. You can also restart animations from the beginning, or disable animations by default.",
+  "main": "index.js",
+  "repository": "git@github.com:simonlindholm/toggle-gifs.git",
+  "author": "Simon Lindholm",
+  "license": "CC0-1.0",
+  "scripts": {
+    "prettier": "prettier --write **/*.{js,json}",
+    "eslint": "eslint --ignore-path .gitignore --fix **/*.js",
+    "test": "jest",
+    "tdd": "yarn test --watch",
+    "extension": "web-ext -s ./extension",
+    "dev:web": "webpack --watch --mode development",
+    "dev:ext": "yarn extension run --url ./example/gif.html",
```

Comment:
let's add `--pref=javascript.options.strict=false` to this (strict warnings are useless and have too many false positive)